src/main.rs

```diff
@@ -7,7 +7,7 @@ use actix_web::{
 };
 use log::info;
 
-const MAX_PAYLOAD_SIZE: usize = 80000000; // 10 Megabyte
+const MAX_PAYLOAD_SIZE: usize = 10000000; // 10 Megabyte
```

Comment:
ok.. this is hard. Should this be 10485760? nobody cares too much.

Reply:
Could this be a configurable parameter, with a default of 10MB? We usually configure things like this through environment variables

Reply:
Yes! Adding support for env variables with PR #9.